_includes/download/board.html

```diff
@@ -160,7 +160,62 @@ <h3>Past Releases</h3>
 {% assign bootloader_id = bootloader_board.bootloader_id %}
 {% if bootloader_version and bootloader_id and bootloader_board %}
 <div class="section unrecommended">
-  <h3>UF2 Bootloader</h3>
+  <h3>Install/Repair UF2 Bootloader</h3>
+  <p>
+    Latest version: {{ bootloader_version }}
+  </p>
+  <p>
+    <strong>The bootloader allows you to load CircuitPython, Makecode, and Arduino programs.
+      The bootloader is not CircuitPython.</strong>
+  </p>
+  <p>
+    It is not necessary to reinstall the bootloader unless a BOOT drive is not visible when in UF2 bootloader mode (FTHRS2BOOT, MAGTAGBOOT, HOUSEBOOT, etc.), in which case the UF2 bootloader may need to be repaired.
+  </p>
```

Comment:
Move this paragraph down too into the ESP32-S2 section, because if the bootloader doesn't show on another board, it's much harder to re-flash it (you may need a J-Link).

Basically, on the ESP32-S2, the bootloader is a "extra" thing, but on most other boards, it is permanently burned on the chip and is essential to its normal operation. So if it didn't appear, there is something really seriously wrong.